docs/pages/zero-trust-access/zero-trust-access.mdx

```diff
@@ -1,6 +1,289 @@
+        {
+          name: "Revoking Access",
+          href: "./management/security/revoking-access/",
+        },
+      ]
+    }
+  ]}
+/>
+
+<UseCasesList
+  title="Use Cases"
+  useCases={[
+    {
+      title: "Enroll and Protect Your Infrastructure",
+      description: "Apps, servers, databases, Kubernetes, desktops, & more",
+      href: "../enroll-resources/",
+    },  
+    {
+      title: "VNet: Build without VPNs",
+      description: "Secure app & SSH access with no VPNs or proxies",
+      href: "../enroll-resources/application-access/guides/vnet/",
+    },
+    {
+      title: "Secure MCP (Protect the Vibes)",
+      description: "Secure MCP integration with granular audit trail",
+      href: "../connect-your-client/model-context-protocol/",
+    },  
+    {
+      title: "Role-Based Access Control (RBAC)",
+      description: "Govern infrastructure access with granular permissions",
+      href: "./rbac-get-started/",
+    },
+    {
+      title: "Passwordless Authentication",
+      description: "Log in securely using biometrics",
+      href: "./authentication/passwordless/",
+    },
+    {
+      title: "Integrate with SSO Providers",
+      description: "Connect Okta, Entra ID, Google, and more",
+      href: "./sso/",
+    },  
+    {
+      title: "Structured Audit Export",
+      description: "Forward audit logs to SIEMs like Splunk and Datadog",
+      href: "./export-audit-events/",
+    },
+    {
+      title: "Identity-Based Audit Events",
+      description: "Detailed audit logs for every user action",
+      href: "../reference/deployment/monitoring/audit/",
+    },  
+    {
+      title: "Session Recording and Playback",
+      description: "Record a detailed review of what took place",
+      href: "../reference/agent-services/desktop-access-reference/sessions/",
+    },
+    {
+      title: "Session Sharing and Moderation",
+      description: "Require a moderator for privileged sessions",
+      href: "./authentication/joining-sessions/",
+    },
+    {
+      title: "Dual Authorization Capabilities",
+      description: "Require approvals to perform critical actions",
+      href: "../identity-governance/access-requests/",
+    },
+    {
+      title: "Manage Clusters with IaC",
+      description: "Create, update, and manage Teleport in declarative code.",
+      href: "./infrastructure-as-code/",
+    },
+  ]}
+/>
+
+{/* vale messaging.protocol-products = NO */}
+<Resources
+  resources={[
+    {
+      title: 'Applications',
+      description: 'Protect web apps, TCP apps, and Cloud APIs',
+      iconComponent: applicationSvg,
+      href: '../enroll-resources/application-access/'
+    },
+    {
+      title: 'Linux Servers',
+      description: 'Secure Linux servers and OpenSSH servers',
+      iconComponent: linuxServersSvg,
+      href: '../enroll-resources/server-access/'
+    },
+    {
+      title: 'Database Access',
+      description: 'AWS, Azure, Google Cloud, cloud DB platforms, & more',
```

Comment:
I'd mention some of our most popular databases in the description instead of mentioning the most popular cloud providers (think Postgres, MongoDB, SQL Server, etc)